packages/logs/src/domain/logsCollection/networkError/networkErrorCollection.ts

```diff
@@ -102,8 +102,14 @@ export function computeFetchResponseText(
   configuration: LogsConfiguration,
   callback: (responseText?: string) => void
 ) {
-  if (!response.body || response.bodyUsed) {
-    // if the body is null or already used by another instrumentation, let's not try to read it.
+  let clonedResponse: Response | undefined
+  try {
+    clonedResponse = response.clone()
+  } catch (e) {
+    // clone can throw if the response has already been used by another instrumentation or is disturbed
+  }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could use a function for that, to avoid repeating the implementation and documentation.